bring_api/bring.py

```diff
@@ -244,6 +267,18 @@ async def getItems(self, listUuid: str) -> BringItemsResponse:
                             if key in BringItemsResponse.__annotations__
                         },
                     )
+
+                    for item in data["purchase"]:
+                        item["itemId"] = await self.translate(
+                            item["itemId"],
+                            to_locale="de-DE",
+                        )
+                    for item in data["recently"]:
+                        item["itemId"] = await self.translate(
+                            item["itemId"],
+                            to_locale="de-DE",
+                        )
+

```

Comment:
is there a better and more efficient way to traverse the items in the BringItemsResponse?